presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -167,6 +167,7 @@ Property Name                                      Description
 
 ``security.config-file``                           Path of config file to use when ``hive.security=file``.
                                                    See :ref:`hive-file-based-authorization` for details.
+``hive.non-managed-table-writes-enabled``          Enable writes to non-managed Hive tables.                    ``false``
```

Comment:
Add a newline before this. All the table entries are separated with blank lines.